app/content/tests/test_strike_model.py

```diff
@@ -1,93 +1,126 @@
 from datetime import datetime, timedelta
 from unittest import mock
 
-from app.content.factories.strike_factory import StrikeFactory
+import pytest
+
+from app.content.factories import StrikeFactory
 from app.util.utils import getTimezone
 
 
+@pytest.mark.django_db
 @mock.patch("app.content.models.strike.now")
```

Comment:
<img width="261" alt="image" src="https://user-images.githubusercontent.com/31009729/163411226-36416f7a-43e3-4a2c-bd43-135452ffc181.png">
```suggestion
@mock.patch("app.util.utils.now")
```

Det er vel egentlig teknisk sett `app.util.utils.now` som mockes?

Reply:
Da skjer det faktisk en feil. For da bruker ikke strike mocken. 
[<img width="551" alt="image" src="https://user-images.githubusercontent.com/70779496/163458092-37ff08c7-3072-42ed-8101-90c56f0b0014.png">](https://write.agrevolution.in/python-unit-testing-mock-and-patch-8ba9c796c9c2)
 Fra: https://write.agrevolution.in/python-unit-testing-mock-and-patch-8ba9c796c9c2

Reply:
Jøss, det visste jeg ikke 👍